samples/nrf_rpc/protocols_serialization/server/README.rst

```diff
@@ -0,0 +1,265 @@
+   :local:
+   :depth: 2
+
+The Protocols serialization server sample runs full Bluetooth® LE and OpenThread stacks, and it exposes selected functions from these stacks over UART.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. table-from-sample-yaml::
+
+To test the sample, you also need another device running the :ref:`Protocols serialization client <nrf_rpc_protocols_serialization_client>` sample.
+
+For testing the Bluetooth LE API serialization, you also need to have the `nRF Connect for Mobile`_ app installed on your smartphone or tablet.
+
+Overview
+********
+
+The Protocols serialization server sample implements the server part of the serialization of OpenThread and Bluetooth LE API calls between two devices that communicate with each other using the UART interface.
+The sample uses the :ref:`nrfxlib:nrf_rpc` and CBOR encoding to serialize function calls.
+
+Configuration
+*************
+
+|config|
+
+Snippets
+========
+
+.. |snippet| replace:: :makevar:`server_SNIPPET`
+
+The following snippets are available:
+
+* ``ble`` - Enables the server part of the :ref:`Bluetooth LE RPC <ble_rpc>`.
+  Also enables the :ref:`Bluetooth LE Nordic UART Service <nus_service_readme>`.
+* ``coex`` - Enables the :ref:`MPSL software coexistence <nrfxlib:mpsl_cx>` implementation on the server device.
+* ``debug`` - Enables debugging the sample by enabling :c:func:`__ASSERT()` statements globally and verbose logging.
+* ``log_rpc`` - Enables logging over RPC.
+* ``openthread`` - Enables the server part of the OpenThread RPC.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf_rpc/protocols_serialization/server`
+
+.. include:: /includes/build_and_run.txt
+
+You can modify the list of enabled features, which by default includes Bluetooth LE support and debug logs.
+
+.. _protocols_serialization_server_sample_testing:
+
+Testing
+*******
+
+After building the Protocols serialization server sample and programming it to your development kit, connect it to a second device running the :ref:`Protocol serialization client <nrf_rpc_protocols_serialization_client>` sample to test either the Bluetooth LE or OpenThread functionality.
+
+.. _protocols_serialization_server_app_connection:
+
+Connecting the client and server samples
+========================================
+
+The client and server devices are connected using two UART peripherals.
+In the protocols serialization samples, one peripheral is used for shell and logging purposes, similarly to other applications and samples, while the other peripheral is used for sending OpenThread and Bluetooth LE remote procedure calls (RPCs).
+
+.. tabs::
+
+    .. group-tab:: nRF52840 SoC
+
+        By default, the nRF52840 system-on-chip uses the ``uart0`` peripheral for shell and logging purposes, and the ``uart1`` peripheral for sending OpenThread and Bluetooth remove procedure calls (RPCs).
```

Comment:
```suggestion
    .. group-tab:: nRF52840 DK

        By default, the nRF52840 DK uses the ``uart0`` peripheral for shell and logging purposes, and the ``uart1`` peripheral for sending OpenThread and Bluetooth remove procedure calls (RPCs).
```
We're showing pictures of the DK, so might as well call it the DK instead of SoC.